### PR TITLE
KAFKA-15740: KRaft support in DeleteOffsetsConsumerGroupCommandIntegrationTest

### PR DESCRIPTION
**Pull request by linzihao1999**
The tests in DeleteOffsetsConsumerGroupCommandIntegrationTest in core/src/test/scala/unit/kafka/admin/DeleteOffsetsConsumerGroupCommandIntegrationTest.scala update to support KRaft.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: core/src/test/scala/unit/kafka/admin/DeleteOffsetsConsumerGroupCommandIntegrationTest.scala [hunk 1]
```diff
@@ -20,7 +20,7 @@ package kafka.admin
 import java.util.Properties
 
 import kafka.server.Defaults
-import kafka.utils.TestUtils
+import kafka.utils.{TestInfoUtils, TestUtils}
```
Comment (ex172000) [inline core/src/test/scala/unit/kafka/admin/DeleteOffsetsConsumerGroupCommandIntegrationTest.scala]: Do we want to put it into a new line?

**Review by ex172000 — APPROVED** [review]

File: core/src/test/scala/unit/kafka/admin/DeleteOffsetsConsumerGroupCommandIntegrationTest.scala (hunk 1 above)
Comment (linzihao1999) [reply core/src/test/scala/unit/kafka/admin/DeleteOffsetsConsumerGroupCommandIntegrationTest.scala]: yes, more pretty

**Comment by linzihao1999** [comment]
Hi @cmccabe @jsancio 

The CI-builds finished and I found no failure test report about DeleteOffsetsConsumerGroupCommandIntegrationTest.
Is there more I need to do before code review. PTAL

https://ci-builds.apache.org/job/Kafka/job/kafka-pr/job/PR-14669/2//testReport/

**Comment by linzihao1999** [comment]
Hi @mimaison PTAL

**Review by mimaison — APPROVED** [review]
LGTM, thanks

**Pull request merged**